wiki/en/Running-a-Server.md

```diff
@@ -125,185 +148,176 @@ To register with one of the Directories built into the Jamulus Client, replace `
+**Note:** This feature is deprecated, and may disappear in a future release.
 
-Run `sudo systemctl daemon-reload` to register them for first use.
+##### `-P or --delaypan` 
+Start with delay panning enabled. This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal “volume” panning.
 
-Now you can run these with the `systemctl` command, for example:
+##### `-s` or `--server`
+Start Jamulus in Server mode
 
-`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)
+##### `--serverbindip` 
+Specify the IP address the Jamulus process will bind to. 
 
-You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
+Normally, Jamulus will listen on all IP addresses on the host machine.  Where the host has multiple network addresses, this option allows one of the addresses to be chosen.
 
+##### `-T or --multithreading` 
+Use multithreading to make better use of multi-core CPUs. This setting may help the Server support more Clients. See also `--numchannels`
 
-## Servers on the desktop
+##### `-u or --numchannels` 
+Maximum number of channels (Clients)
 
-Jamulus can be run in Server mode from the desktop. This gives you a graphical user interface to control most of the settings.
+##### `-z or --startminimized` 
+Start the Jamulus Server graphical user interface in the minimized window state.
 
-* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return.
+### Other options
 
+{% include_relative Include-Shared-Commands.md %}
 
-## Server Setup
+---
 
-<figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus Server setup window"></figure>
+## Recording
 
-### The Directory list
+##### `-R or --recording` 
+Set server recording directory. By default, the Server will record when a session is active. 
 
-**None**: By default, you will not be connected to a Directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your Server in this mode.
+**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (but be aware that doing could be a security risk).
```

Comment:
```suggestion
**Note:** You will need to save recordings to a path _outside_ of the jamulus home directory, or remove `ProtectHome=true` from your systemd unit file, but be aware that doing could be a security risk.
```

Reply:
is the default directory `jamulus` with no capitalization?

Reply:
That's down to the contributed `.deb` installer.